saml-core/src/main/java/org/keycloak/saml/processing/core/parsers/saml/metadata/SAMLAttributeParser.java

```diff
@@ -16,20 +16,12 @@
  */
 package org.keycloak.saml.processing.core.parsers.saml.metadata;
 
-import org.keycloak.dom.saml.v2.assertion.AttributeType;
-import org.keycloak.saml.common.exceptions.ParsingException;
-import org.keycloak.saml.common.util.StaxParserUtil;
-import org.keycloak.saml.processing.core.parsers.saml.assertion.SAMLAttributeValueParser;
-
-import javax.xml.stream.XMLEventReader;
-import javax.xml.stream.events.StartElement;
-
 /**
  * Parse the <conditions> in the saml assertion
  *
  * @since Oct 14, 2010
  */
-public class SAMLAttributeParser extends AbstractStaxSamlMetadataParser<AttributeType> {
+public class SAMLAttributeParser extends SAMLAttributeTypeParser {
```

Comment:
Remove and use parser from `org.keycloak.saml.processing.core.parsers.saml.assertion`

Reply:
org.keycloak.saml.processing.core.parsers.saml.assertion.SAMLAttributeParser  will be used in this PR. This SAMLAttributeParser existed in code and it was not used. Do you want to keep org.keycloak.saml.processing.core.parsers.saml.metadata.SAMLAttributeParser same as Keycloak master or to remove it?